explore/circadian-efficacy/README.md

```diff
@@ -10,3 +10,25 @@ Evaluating unsupervised DWPC p-values and path contributions to predict whether
 + [`downloads/aat8806_Data_file_S1.xlsx`](downloads/aat8806_Data_file_S1.xlsx) contains data from CircaDB, which provides tissue-specific rhythmical scores for human genes. 
   The data was downloaded from Data file S1 of [A database of tissue-specific rhythmically expressed human genes has potential applications in circadian medicine](https://doi.org/10.1126/scitranslmed.aat8806).
   The algorithm 'CYCLOPS', which was developed to score genes based on circadian rhythms, can be accessed from [CYCLOPS reveals human transcriptional rhythms in health and disease](https://doi.org/10.1073/pnas.1619320114).
+
++ [`downloads/drug_links.csv`](downloads/drug_links.csv) contains mapping between compound names and DrugBank IDs, as well as other external links.
+  The data was downloaded from [External Drug Links in Drugbank](https://www.drugbank.ca/releases/5-1-3/downloads/all-drug-links).
+
++ [`downloads/GTEx_Analysis_2016-01-15_v7_RNASeQCv1.1.8_gene_median_tpm.gct`](downloads/GTEx_Analysis_2016-01-15_v7_RNASeQCv1.1.8_gene_median_tpm.gct) contains median expression of all genes in 55 GTEx tissues.
+  The data was downloaded from [GTEx V7 RNA-seq data](https://storage.googleapis.com/gtex_analysis_v7/rna_seq_data/GTEx_Analysis_2016-01-15_v7_RNASeQCv1.1.8_gene_median_tpm.gct.gz)
+
+# Generated datasets
+
++ [`data/disease_doid.tsv`](data/disease_doid.tsv) contains manually curated mapping from therapeutic area to disease ontology ID.
+
++ [`data/tissue_uberon.tsv`](data/tissue_uberon.tsv) contains manually curated mapping from CircaDB tissue terms to exact Uberon tissue IDs, hetionet Uberon tissue IDs(closest tissue available in hetionet), and GTEx tissue names.
```

Comment:
Shouldn't we also have a manual `compound_map.tsv`? For example, I noticed "Folinic Acid" does not get mapped to `DB00650`?